core/src/main/java/org/infinispan/remoting/transport/Transport.java

```diff
@@ -248,6 +263,6 @@
    @Experimental
    <T> CompletionStage<T> invokeCommands(Collection<Address> targets,
                                          Function<Address, ReplicableCommand> commandGenerator,
-                                         ResponseCollector<T> responseCollector, long timeout,
-                                         DeliverOrder deliverOrder);
+                                         ResponseCollector<T> collector, DeliverOrder deliverOrder,
```

Comment:
hmm isn't the `Transport` kind of public API? you can set it via `TransportConfigurationBuilder.transport(Transport);`

Reply:
I was thinking that I've protected myself against that by marking the method `@Experimental` when I first added it, but you're right, I should have also made it `default`.

Reply:
ah... I didn't see the annotation :)